examples/resources/dsm_csr/resource.tf

```diff
@@ -1,18 +1,18 @@
-// To use this resource, it is required to create a custom plugin in DSM first.
-// Copy the plugin from https://github.com/fortanix/terraform-provider-dsm/blob/main/plugins/Terraform-Plugin-CSR.lua
-// Create the custom plugin in DSM 
-// Plugin title: "Terraform Plugin - CSR"
+# To use this resource, it is required to create a custom plugin in DSM first.
+# Copy the plugin from https:#github.com/fortanix/terraform-provider-dsm/blob/main/plugins/Terraform-Plugin-CSR.lua
```

Comment:
The URL is invalid here due to the replacement of `//` as `#`

Reply:
Updated.